testsuite/cluster-test/src/cluster.rs

```diff
@@ -45,8 +47,14 @@ impl Cluster {
                 )
             })
             .collect();
-        let mint_key: Ed25519PrivateKey = generate_key::load_key(mint_file);
-        let mint_key_pair = KeyPair::from(mint_key);
+
+        let mint_key_pair = if premainnet {
+            let keypair: KeyPair<Ed25519PrivateKey, Ed25519PublicKey> =
```

Comment:
So generating random key here makes no sense to be honest

Perhaps we should just make mint key to be an Option?

Reply:
yea, my thinking is use random key here to reserve a place, and will make another PR to make this to be Option since there are quite a few places and apis need to change